FeedStoreChallenge/Core Data/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by YH Kung on 2021/7/8.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal final class ManagedCache: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`. Please remove it from this class and its extension.
```suggestion
final class ManagedCache: NSManagedObject {
```